diff --git a/suricata.yaml.in b/suricata.yaml.in
index e6bd26d6a2..75cf8a9d6a 100644
--- a/suricata.yaml.in
+++ b/suricata.yaml.in
@@ -1955,7 +1955,7 @@ profiling:
 
     # Profiling can be disabled here, but it will still have a
     # performance impact if compiled in.
-    enabled: yes
+    enabled: no
     filename: rule_perf.log
     append: yes
     # Set active to yes to enable rules profiling at start
